app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -46,7 +46,7 @@
     <h3 class="heading-l">Can I get help funding my training?</h3>
 
     <p>
-      If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship, which doesn't need to be paid back, to support you while you are training.
+      If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free, which doesn't need to be paid back, to support you while you are training.
```

Comment:
@emmaachesongray I know this is how we worded it in prod previously but it is worth making it slightly clearer?

```suggestion
      If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>, which doesn't need to be paid back, to support you while you are training.
```

Reply:
(or use the wording on `/landing/how-to-become-a-teacher`)

Reply:
```suggestion
      If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free. This is to support you while you're training and does not need to be paid back.
```